usb/device/targets/TARGET_STM/USBPhy_STM32.cpp

```diff
@@ -302,7 +322,34 @@ void USBPhyHw::init(USBPhyEvents *events)
     hpcd.State = HAL_PCD_STATE_RESET;
     HAL_PCD_Init(&hpcd);
 
-
+#if defined(TARGET_NUCLEO_L073RZ)
+    // EP0
+    HAL_PCDEx_PMAConfig(&hpcd, 0x00, PCD_SNG_BUF, 0x30);
+    HAL_PCDEx_PMAConfig(&hpcd, 0x80, PCD_SNG_BUF, 0x70);
+    // EP1
+    HAL_PCDEx_PMAConfig(&hpcd, 0x01, PCD_SNG_BUF, 0xB0);
+    HAL_PCDEx_PMAConfig(&hpcd, 0x81, PCD_SNG_BUF, 0xF0);
+    // EP2
+    HAL_PCDEx_PMAConfig(&hpcd, 0x02, PCD_SNG_BUF, 0x130);
+    HAL_PCDEx_PMAConfig(&hpcd, 0x82, PCD_SNG_BUF, 0x170);
+    // EP3
+    HAL_PCDEx_PMAConfig(&hpcd, 0x03, PCD_DBL_BUF, 0x01F001B0);
+    HAL_PCDEx_PMAConfig(&hpcd, 0x83, PCD_SNG_BUF, 0x230);
+
+
+//    // EP0
+//    HAL_PCDEx_PMAConfig(&hpcd, 0x00, PCD_SNG_BUF, 0x30);
+//    HAL_PCDEx_PMAConfig(&hpcd, 0x80, PCD_SNG_BUF, 0x70);
```

Comment:
I think Cruz will request you to remove dead code :-)

Reply:
Please and thank you!